### ISSUE TITLE
ARM Support / dotnet 7 target

### ISSUE THREAD

**Issue by jufa2401** [skarpdev/dotnet-version-cli#100]
With usage in M1 Macs the cli complains over architecture:

Below is a sample error found on my machine

```

╰─ dotnet version prerelease
Failed to load /usr/local/share/dotnet/host/fxr/7.0.0/libhostfxr.dylib, error: dlopen(/usr/local/share/dotnet/host/fxr/7.0.0/libhostfxr.dylib, 0x0001): tried: '/usr/local/share/dotnet/host/fxr/7.0.0/libhostfxr.dylib' (mach-o file, but is an incompatible architecture (have '**arm64**', need '**x86_64**')), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/share/dotnet/host/fxr/7.0.0/libhostfxr.dylib' (no such file), '/usr/local/share/dotnet/host/fxr/7.0.0/libhostfxr.dylib' (mach-o file, but is an incompatible architecture (have '**arm64**', need '**x86_64**'))
The library libhostfxr.dylib was found, but loading it from /usr/local/share/dotnet/host/fxr/7.0.0/libhostfxr.dylib failed
  - Installing .NET prerequisites might help resolve this problem.
     https://go.microsoft.com/fwlink/?linkid=2063366


```

**Comment by nover** [comment]
@jufa2401 this seems more like a dotnet tooling issue - you have donet 7 installed, but we haven't built it for that target yet. I'll have a look at that, but in the meantime could you try the same with dotnet 6 lts installed? 😊

**Comment by AlbertoMonteiro** [comment]
@jufa2401 for me, the issue was 
![image](https://user-images.githubusercontent.com/836496/232928125-81196eb7-3d33-45ac-873d-301d4a8c3419.png)

```
You must install or update .NET to run this application.
App: /root/.dotnet/tools/dotnet-version
Architecture: x64
Framework: 'Microsoft.NETCore.App', version '6.0.0' (x64)
.NET location: /usr/share/dotnet
The following frameworks were found:
  7.0.5 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]
Learn about framework resolution:
https://aka.ms/dotnet/app-launch-failed
To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=6.0.0&arch=x64&rid=debian.11-x64
```

I "fixed" it by defining the following environment variable

```
export DOTNET_ROLL_FORWARD=LatestMajor
```

**Comment by nover** [comment]
@AlbertoMonteiro thanks for the workaround.

Issue has been fixed via #101 and is available as version `2.4.1` on nuget.
